app/src/main/java/sdp/moneyrun/map/MapActivity.java

```diff
@@ -194,14 +201,21 @@ public void removeCoin(Coin coin){
     }
 
 
-    public void moveCameraTo(float latitude, float longitude){
+    public void moveCameraTo(float latitude, float longitude, double zoom){
         CameraPosition position = new CameraPosition.Builder()
                 .target(new LatLng(latitude, longitude))
+                .zoom(zoom)
                 .build();
         mapboxMap.animateCamera(CameraUpdateFactory.newCameraPosition(position));
     }
 
-
+    public void moveCameraWithoutAnimation(double latitude, double longitude, double zoom){
```

Comment:
I don't know if the `targe`t and the` zoom` function check for validity of the arguments ( check if lat, long, less than zero, zoom is acceptable) if the checks in those functions are appropriate for our case everything is fine, otherwise maybe add a check for `latitude, longitude, and zoom`